source/expressions.tex

```diff
@@ -1980,7 +1979,7 @@
 \grammarterm{capture-default} that is \tcode{=}, each
 \grammarterm{simple-capture} of that \grammarterm{lambda-capture} shall
 be of the form
-``\tcode{\&} \grammarterm{identifier}'',
+``\tcode{\&} \opt{\tcode{...}} \grammarterm{identifier}'',
```

Comment:
`...` on the wrong side?

Reply:
This one is on the correct (and right 😉 ) side. But the grammar is wrong, sigh.

Reply:
God I am just the very worst at this. And then I confused myself enough to leave the totally wrong comment about this. 

This PR matches the paper, but the paper is wrong... the `...` should definitely be on the right side of the _`identifier`_. This makes me very sad. I'm going to go sit in a corner and think about what I did.

In the meantime, can we just Do The Right Thing ™️  and move the `...`?

Reply:
While technically a normative change, I think this was an obvious mistake; fixed editorially.